### PR TITLE
Kernel to Kernel Streaming Support in Profile Summary

### PR DESCRIPTION
**Pull request by hackwa**

### CODE REVIEW

**Comment by sonals** [comment]
ok to test

**Comment by hackwa** [comment]
retest this please.

File: src/runtime_src/driver/xclng/tools/xbutil/xbutil_debug.cpp [hunk 1]
```diff
@@ -147,9 +166,9 @@ int xcldev::device::readSPMCounters() {
                   << "  " << std::setw(16) << debugResults.ReadTranx[i]
                   << "  " << std::setw(16) << debugResults.OutStandCnts[i]
                   << "  " << std::hex << "0x" << std::setw(14) << debugResults.LastWriteAddr[i] << std::dec
```
Comment (pgschuey) [inline src/runtime_src/driver/xclng/tools/xbutil/xbutil_debug.cpp]: I just noticed - we can remove "<< std::dec" at the end of lines 168 and 170

**Comment by hackwa** [comment]
retest this please

**Comment by stsoe** [comment]
retest this please

**Comment by sonals** [comment]
@hackwa , can you make the const correctness changes identified by Soren?

**Comment by hackwa** [comment]
Thanks! Added const @sonals @stsoe

**Pull request merged**